### PR TITLE
Console sprint76

### PR DESCRIPTION
**Pull request by jdgemm**

### CODE REVIEW

File: docs/use-the-network/console-marketplace/console-marketplace.mdx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+- Add new integration controller action to ```/lib/console_web/controllers/v1/channel_controller.ex``` under ```create_community_channel()``` or create a new function if needed.
+
+More info about Integrations [here](/use-the-network/console/integrations).
+
+## Enable Marketplace Integrations (For Console Hosting Providers)
+
+:::warning
+
+An Integration must be enabled for devices to transfer data throught it. For example, if an existing Integration is disabled, any devices will no longer be able to transfer data via that Integration.
```
Comment (vicmgs) [inline docs/use-the-network/console-marketplace/console-marketplace.mdx]: through*

File: docs/use-the-network/console-marketplace/console-marketplace.mdx [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+:::
+
+Console Hosting Providers can choose which Integrations are enabled for users by making the following configuration settings. 
+
+More information about becoming a Console Hosting Provider [here](/use-the-network/run-a-network-server). 
+
+Set ```ALLOWED_INTEGRATIONS.env``` variable to toggle desired integrations.
```
Comment (vicmgs) [inline docs/use-the-network/console-marketplace/console-marketplace.mdx]: move .env out of the quotes

File: docs/use-the-network/console-marketplace/console-marketplace.mdx [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+- To enable all reviewed integrations (default): ```ALLOWED_INTEGRATIONS=all```.
+
+- To enable selected integrations: ```ALLOWED_INTEGRATIONS=<integrations>```. For example, to enable the following integrations: HTTP, MQTT, CARGO: ```ALLOWED_INTEGRATIONS=http,mqtt,cargo```
+
+Integration keys can be found in ```/templates/allowed-integrations.json```.
```
Comment (vicmgs) [inline docs/use-the-network/console-marketplace/console-marketplace.mdx]: check the github issue again, i changed the file path

File: docs/use-the-network/console/integrations/integrations.mdx [hunk 4]
```diff
@@ -11,18 +11,33 @@ import useBaseUrl from "@docusaurus/useBaseUrl";
...
 
+Community Integrations have been submitted to the Console Marketplace, approved by the Helium Foundation and chosen by the Console Hosting Provider. 
+
+To learn more about the Console Marketplace and submitting an Integration, go [here](/use-the-network/console-marketplace).
+
+If an Integration is not available please contact your provider. By default, the Helium Foundation Console will enable all approved Integrations. 
+
+:::important
+
+Pre-Marketplace Integrations will continue to work as expected. However, devices or groups will need to connect to an Integration to benefit from future updates provided by the Integration developer.
```
Comment (vicmgs) [inline docs/use-the-network/console/integrations/integrations.mdx]: connect to a new community Integration

File: docs/use-the-network/console-marketplace/console-marketplace.mdx (hunk 1 above)
Comment (jdgemm) [reply docs/use-the-network/console-marketplace/console-marketplace.mdx]: fixed

**Review by kent-williams — APPROVED** [review]

**Pull request merged**